emails/tests/models_tests.py

```diff
@@ -441,6 +444,18 @@ def test_has_premium_with_unlimited_subsription_returns_True(self):
         premium_profile = baker.make(Profile, user=premium_user)
         assert premium_profile.has_premium is True
 
+    def test_has_phone_with_unlimited_subsription_returns_True(self):
```

Comment:
Removed in e4f493fe9123f844f733797911f1c01547b82ff2. Leaving `test_has_phone_default_False` function.